### PR TITLE
LPS-102177 Create new exception mappers

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

File: modules/apps/segments/segments-asah-rest-impl/src/main/java/com/liferay/segments/asah/rest/internal/jaxrs/exception/mapper/LockedSegmentsExperimentExceptionMapper.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import com.liferay.segments.exception.LockedSegmentsExperimentException;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * Converts any {@code LockedSegmentsExperimentException} to a {@code 400} error.
```
Comment (nhpatt) [inline modules/apps/segments/segments-asah-rest-impl/src/main/java/com/liferay/segments/asah/rest/internal/jaxrs/exception/mapper/LockedSegmentsExperimentExceptionMapper.java]: @review

File: modules/apps/segments/segments-asah-rest-impl/src/main/java/com/liferay/segments/asah/rest/internal/jaxrs/exception/mapper/SegmentsExperimentRelSplitExceptionMapper.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import com.liferay.segments.exception.SegmentsExperimentRelSplitException;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * Converts any {@code SegmentsExperimentRelSplitException} to a {@code 400} error.
```
Comment (nhpatt) [inline modules/apps/segments/segments-asah-rest-impl/src/main/java/com/liferay/segments/asah/rest/internal/jaxrs/exception/mapper/SegmentsExperimentRelSplitExceptionMapper.java]: @review

File: modules/apps/segments/segments-asah-rest-impl/src/main/java/com/liferay/segments/asah/rest/internal/jaxrs/exception/mapper/SegmentsExperimentStatusExceptionMapper.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import com.liferay.segments.exception.SegmentsExperimentStatusException;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * Converts any {@code SegmentsExperimentStatusException} to a {@code 400} error.
```
Comment (nhpatt) [inline modules/apps/segments/segments-asah-rest-impl/src/main/java/com/liferay/segments/asah/rest/internal/jaxrs/exception/mapper/SegmentsExperimentStatusExceptionMapper.java]: @-..

**Comment by dgarciasarai** [comment]
Ouch!  😅

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by nhpatt** [comment]
semver :P

**Pull request merged**